src/main/java/uk/gov/hmcts/reform/coh/domain/Question.java

```diff
@@ -34,6 +34,12 @@
     @Column(name = "question_round")
     private Integer questionRound;
 
+    @ElementCollection
```

Comment:
@ElementCollection is a simpler way to implement parent/child relationship. The table is only 2 columns and one of those is the parent question id and there's no need to model the question entity. We just need to return the linked question id to JUI and they'll do the URL linking